File: app/src/main/java/com/jnjcomu/edison/model/Region.java
package com.jnjcomu.edison.model;

import com.google.gson.annotations.SerializedName;

/**
 * @author CodeRi13 <ruto1924@gmail.com>
 * @since 2017-05-12
 */

public class Region {
    @SerializedName("region") int regionID;
    @SerializedName("name") String placeName;

    public Region(int regionID, String placeName) {
        this.regionID = regionID;
        this.placeName = placeName;
    }

    public int getRegionID() {
        return regionID;
    }

    public void setRegionID(int regionID) {
        this.regionID = regionID;
    }

    public String getPlaceName() {
        return placeName;
    }

    public void setPlaceName(String placeName) {
        this.placeName = placeName;
    }
}

File: app/src/main/java/com/jnjcomu/edison/model/Region.kt
package com.jnjcomu.edison.model

import com.google.gson.annotations.SerializedName

/**
 * @author CodeRi13 <ruto1924></ruto1924>@gmail.com>
 * *
 * @since 2017-05-12
 */

class Region(@SerializedName("region") var regionID: Int, @SerializedName("name") var placeName: String)
